api/converter_utils.go

```diff
@@ -210,27 +193,27 @@ func lsigToTransactionLsig(lsig sdk_types.LogicSig) *generated.TransactionSignat
 	return &ret
 }
 
-func onCompletionToTransactionOnCompletion(oc sdk_types.OnCompletion) generated.OnCompletion {
+func onCompletionToTransactionOnCompletion(oc transactions.OnCompletion) generated.OnCompletion {
 	switch oc {
-	case sdk_types.NoOpOC:
+	case transactions.NoOpOC:
 		return "noop"
-	case sdk_types.OptInOC:
+	case transactions.OptInOC:
 		return "optin"
-	case sdk_types.CloseOutOC:
+	case transactions.CloseOutOC:
 		return "closeout"
-	case sdk_types.ClearStateOC:
+	case transactions.ClearStateOC:
 		return "clear"
-	case sdk_types.UpdateApplicationOC:
+	case transactions.UpdateApplicationOC:
 		return "update"
-	case sdk_types.DeleteApplicationOC:
+	case transactions.DeleteApplicationOC:
 		return "delete"
 	}
 	return "unknown"
```

Comment:
this would probably be better implemented in a map.. but only if it would remain after the rest of the upcoming changes.

Reply:
This is unrelated to accounting changes, but we can address it later.